Sources/CocoaLumberjackSwiftLogBackend/DDLogHandler.swift

```diff
@@ -135,12 +142,14 @@ public struct DDLogHandler: LogHandler {
     @usableFromInline
     var loggerInfo: LoggerInfo
 
+    @inlinable
     public var logLevel: Logger.Level {
-        get { loggerInfo.logLevel }
+        get { return loggerInfo.logLevel }
         set { loggerInfo.logLevel = newValue }
     }
+    @inlinable
     public var metadata: Logger.Metadata {
-        get { loggerInfo.metadata }
+        get { return loggerInfo.metadata }
```

Comment:
We would have actually had a problem building this module against Swift versions pre 5.2 (or 5.1?) due to those missing returns. 🙈 

Either way, I guess we can soon drop support for those Swift versions and use those nice implicit returns...